src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlStream.cs

```diff
@@ -461,13 +407,17 @@ override public int Read(byte[] buffer, int offset, int count)
             return intCount;
         }
 
-        override public long Seek(long offset, SeekOrigin origin)
+        public override long Seek(long offset, SeekOrigin origin)
         {
             long pos = 0;
 
             if (null == _cachedBytes)
             {
-                throw ADP.StreamClosed();
+                throw ADP.StreamClosed(
```

Comment:
AdapterUtil has been merged and if it uses the `[CallerMemberName] string method = ""` version from core you can remove the netfx conditionals and use allow the method name to be propagated by the compiler for all calls to it in this file.

Reply:
@Wraith2  That was resolved by merging the branch with main branch. Thanks.